go.mod

```diff
@@ -70,6 +70,7 @@ require (
 	k8s.io/utils v0.0.0-20200720150651-0bdb4ca86cbc
 	kubevirt.io/client-go v0.0.0-00010101000000-000000000000
 	kubevirt.io/containerized-data-importer v1.25.0
+	kubevirt.io/controller-lifecycle-operator-sdk v0.1.1
```

Comment:
u need or a leftover?

Reply:
I am not sure. I deleted it and it created it again.

Reply:
i would suggest you to remove just the mount, and run the generate, if you see it still add it, you will know its something in your env,

the only thing that can create it is the test or your local env, from a glance its not the tests (but i might be wrong because i just glanced)

Reply:
removed.